tests/test_game_events.c

```
@@ -451,6 +452,86 @@ TEST(delete_player_anim_roundtrip)
+/* === ObjectExplodingEvent parser (kill signal, PythonEvent factory 0x8129) === */
+
+TEST(exploding_event_weapon_kill)
+{
+    /* Killer (slot 0 ship) destroys victim (slot 1 ship). */
+    i32 killer_id = bc_make_ship_id(0);
+    i32 victim_id = bc_make_ship_id(1);
+
+    u8 wire[64];
+    int wire_len = bc_build_python_exploding_event(wire, sizeof(wire),
+                                                   killer_id, victim_id,
+                                                   killer_id, 9.5f);
```

Clang (14.0.6)

[warning] floating point literal has suffix 'f', which is not uppercase

(readability-uppercase-literal-suffix)